File: 4__Ruby_Program/6__grade_classification.rb
# Input grade and calculate with if statements

print "Enter a grade: "
grade = gets.chomp.to_i

if grade >= 90 and grade <= 100
    cls = 'A'
elsif grade >= 80 and grade <= 100
    cls = 'B'
elsif grade >= 70 and grade <= 100
    cls = 'C'
elsif grade >= 60 and grade <= 100
    cls = 'D'
elsif grade >= 0 and grade <= 100
    cls = 'F'
else
    puts "Please enter correct grade score"
    exit
end

puts "The grade classification is #{cls}"

File: 2__C++_Program/6__grade_classification.cpp
#include <iostream>

using namespace std;

int main() {
    // Input grade and calculate with if statements
    int grade;
    char cls;

    cout << "Enter a grade: "; cin >> grade;

    if (grade >= 90 && grade <= 100) {
        cls = 'A';
    } else if (grade >= 80) {
        cls = 'B';
    } else if (grade >= 70) {
        cls = 'C';
    } else if (grade >= 60) {
        cls = 'D';
    } else {
        cls = 'F';
    }

    cout << "The classification is: " << cls << endl;

    return 0;
}